aioasuswrt/asuswrt.py

```diff
@@ -50,11 +50,6 @@
     r'\s' +
     r'.*')
 
-_IFCONFIG_CMD = 'ifconfig eth0 |grep bytes'
```

Comment:
Tests in `test_regex.py` are failing..

Reply:
That's my bad. I removed obsolete code, but didn't remove the tests that exercise it.
I'll make a new PR to fix that up.

If you guys have access to travis, circleci or github actions I can configure some CI that runs tests on each commit / PR, preventing these issues.